src/Raven.Server/Documents/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections.Generic;
+using JetBrains.Annotations;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.ServerWide;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Web.System.Processors.OngoingTasks;
+using Sparrow.Json;
+using Sparrow.Utils;
+
+namespace Raven.Server.Documents.Handlers.Processors.OngoingTasks;
+
+internal class ShardedOngoingTasksHandlerProcessorForGetOngoingTasks : AbstractOngoingTasksHandlerProcessorForGetOngoingTasks<TransactionOperationContext>
+{
+    [NotNull]
+    private readonly ShardedDatabaseRequestHandler _handler;
+
+    public ShardedOngoingTasksHandlerProcessorForGetOngoingTasks([NotNull] ShardedDatabaseRequestHandler requestHandler, [NotNull] JsonContextPoolBase<TransactionOperationContext> contextPool) : base(requestHandler, contextPool)
```

Comment:
better to remove context pool and passs requestHandler.ContextPool IMO

Reply:
done